src/common/latest/__tests__/prepare-task-test.ts

```diff
@@ -3,16 +3,30 @@ import * as toolLib from "azure-pipelines-tool-lib/tool";
 import { Guid } from "guid-typescript";
 import * as path from "path";
 import { SemVer } from "semver";
-import { scanner } from "../config";
-import { JdkVersionSource, TaskVariables } from "../helpers/constants";
+import { JdkVersionSource } from "../helpers/constants";
 import * as request from "../helpers/request";
+import { AzureTaskLibMock } from "../mocks/AzureTaskLibMock";
+import { AzureToolLibMock } from "../mocks/AzureToolLibMock";
 import * as prept from "../prepare-task";
 import Endpoint, { EndpointType } from "../sonarqube/Endpoint";
 import Scanner, { ScannerCLI, ScannerMSBuild, ScannerMode } from "../sonarqube/Scanner";
 import TaskReport from "../sonarqube/TaskReport";
 
+jest.mock("azure-pipelines-task-lib/task");
+jest.mock("azure-pipelines-tool-lib/tool");
```

Comment:
For some reason, we need to use `jest.mock()` on these libs both in the mock helper and in the utility functions that use them. I'm happy to optimize this if you have any idea about how to, because if we're using the mock helpers, then for sure we want these libs mocked!

Reply:
I think that's fine 👍🏻 we were overriding the method responses below and this will basically catch if we need to mock something specific